ee/ee_core/linkfile

```diff
@@ -93,21 +93,22 @@ SECTIONS {
 		*(.sbss.*)
 		*(.gnu.linkonce.sb*)
 		*(.scommon)
-	} >ram84
+	} >MAIN_REGION
 
 	.bss ALIGN(128) : {
 		*(.bss)
 		*(.bss.*)
 		*(.gnu.linkonce.b*)
 		*(COMMON)
-	} >ram84
+	} >MAIN_REGION
 	_end_bss = .;
 
-	/* Symbols needed by crt0.s.  */
-	PROVIDE(_heap_size = -1);
-	PROVIDE(_stack = .);
-	PROVIDE(_stack_size = 0x00097000 - _stack);	/* Any remaining space up to the module storage region, is available for the stack. */
+	_end = .;
+	end = _end;
```

Comment:
Now I'm curious.... why do we need `end` and `_end`?

Reply:
We only need `_end`. But the sony documentation defines both symbols so thats why we have both.